### DISCUSSION TITLE
[Question]: Gemini 1.5 Pro Maximum Output Token Issue

### DISCUSSION THREAD

**Discussion by sgtalh4381** [danny-avila/LibreChat#2432] [Q&A] [answered]
### What is your question?

Why does the Gemini 1.5 Pro on LibreChat currently only support a maximum output of 2048 tokens. In fact, Gemini 1.5 Pro can support 8192 token outputs.

### More Details

There is no more details.

### What is the main subject of your question?

Endpoints

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/126679241/fd55bd47-e5bc-4232-aada-0cb5430fc92e)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
That is from legacy settings, I will update the max later today. In the future, settings like this will be configurable so I don't need to keep editing the code.

**Reply by sgtalh4381** [reply]
Thank you. Thank you for your excellent work!

**Reply by danny-avila** [reply]
thanks! This should fix it: https://github.com/danny-avila/LibreChat/pull/2434
